certbot_dns_synergy_wholesale.py

```diff
@@ -105,7 +105,26 @@ def __init__(self, reseller_id, api_key) -> None:
             "apiKey": api_key,
         }
 
+    def domainsplitter(self, full_domain):
```

Comment:
```suggestion
    def _parse_domain(self, full_domain):
```

In python I like to use _ to signal that it's a private function